src/DatePicker/DateDisplay.js

```diff
@@ -47,7 +47,7 @@ function getStyles(props, context, state) {
       marginBottom: 10,
     },
     yearTitle: {
-      cursor: (!selectedYear && !props.disableYearSelection) ? 'pointer' : 'default',
+      cursor: props.disableYearSelection ? 'not-allowed' : (!selectedYear ? 'pointer' : 'default'),
```

Comment:
Good point. I agree as the default value of `disableYearSelection` is `false` 👍 .